scripts/output/single/ar6Climate.R

```diff
@@ -0,0 +1,152 @@
+library(madrat)
+library(remind2)
+library(quitte)
+library(piamInterfaces)
+library(yaml)
+library(lucode2)
+library(readxl) # GA: Wont be necessary after https://github.com/iiasa/climate-assessment/pull/43 goes into release
+
+############################# BASIC CONFIGURATION #############################
+gdx_name     <- "fulldata.gdx"             # name of the gdx
+cfg_name     <- "cfg.txt"                  # cfg file for getting file paths
+
+
+if(!exists("source_include")) {
+   # Define arguments that can be read from command line
+   outputdir <- "."
+   readArgs("outputdir", "gdx_name", "gdx_ref_name", "gdx_refpolicycost_name")
+}
+
+gdx           <- file.path(outputdir, gdx_name)
+cfg_path      <- file.path(outputdir, cfg_name)
+logfile       <- file.path(outputdir, "climate.log") # specific log for python steps
+scenario <- getScenNames(outputdir)
+remind_reporting_file <- file.path(outputdir,paste0("REMIND_generic_", scenario,".mif"))
+
+print(getwd())
+############################# PREPARING EMISSIONS INPUT #############################
+
+# Read the cfg to get the location of MAGICC-related files
+cfg <- read_yaml(cfg_path)
+
+# Read the GDX and run reportEmi
+# gdxpath <- "fulldata.gdx"
+cat(date()," ar6Climate.R: Running reportEmi \n")
```

Comment:
Why do we need to rerun reportEmi, why not load all the data from the mif file? We can then add a `yaml` file which contains all the variables MAGICC needs in the format
```
- Emissions|BC:
    unit: Mt BC/yr
- Emissions|BC|AFOLU:
    unit: Mt BC/yr
```
and pass that file that we put somewhere in piamInterfaces or REMIND repo as `iiasatemplate` to `generateIIASASubmission` and you only get back what you need.